src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -73,22 +74,28 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps, DropdownVariables>
     background: listBackgroundColor,
   }),
 
-  emptyListItem: ({ variables: { listItemBackgroundColor } }) => ({
-    backgroundColor: listItemBackgroundColor,
+  loadingMessage: ({ variables: v }): ICSSInJSStyle => ({
+    [`&.${ListItem.className}`]: { backgroundColor: v.listItemBackgroundColor },
```

Comment:
Why do we need this selector here? As far as I can see we are applying the styles on the ListItem component.

Reply:
Removed it, nice catch 👍